HackingWithSwift/week1/day_5/Running throwing functions/Running throwing functions .playground/Contents.swift

```diff
@@ -0,0 +1,31 @@
+enum priceError : Error {
+    case  ETH
+}
+func checkPrice (_ price: Int) throws ->Bool {
+    if price == 5000 {
+        throw priceError.ETH
+    }
+    return true
+}
+do {
+    try checkPrice(5000)
+    print("that price is good!")
+} catch {
+    print("You can't sell for this price")
+}
+enum burgerError : Error {
+    case noMeat
+    case noBread
+}
+func makeBurger (type: String) throws {
+    if type == "Vega burger" {
+        print("Your burger will be ordering in 15 minutes")
+    } else {
+        throw burgerError.noMeat
+    }
+}
+do {
+    try makeBurger(type: "meet burger")
+} catch {
+    print("We don't have meet now")
+}
```

Comment:
Take a look at formatting. + Remember to add blank lines (between code blocks) to make code more readable